shell/platform/windows/window_win32.cc

```diff
@@ -564,6 +593,9 @@ void WindowWin32::Destroy() {
 void WindowWin32::HandleResize(UINT width, UINT height) {
   current_width_ = width;
   current_height_ = height;
+  if (direct_manipulation_owner_) {
```

Comment:
Is it possible for `direct_manipulation_owner_` to be null? If not we should just assert. (Also many other similar cases.)

Reply:
If Direct Manipulation is not supported for some reason it would be null, so I think it should not be an assert here.